plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/completion/weighers/PreferGetSetMethodsToPropertyWeigher.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+package org.jetbrains.kotlin.idea.completion.weighers
+
+import com.intellij.codeInsight.lookup.LookupElement
+import com.intellij.codeInsight.lookup.LookupElementWeigher
+import com.intellij.codeInsight.lookup.WeighingContext
+import com.intellij.openapi.util.Key
+import org.jetbrains.kotlin.idea.frontend.api.KtAnalysisSession
+import org.jetbrains.kotlin.idea.frontend.api.symbols.KtPropertySymbol
+import org.jetbrains.kotlin.idea.frontend.api.symbols.KtSymbol
+import org.jetbrains.kotlin.name.Name
+import org.jetbrains.kotlin.psi.UserDataProperty
+
+internal object PreferGetSetMethodsToPropertyWeigher {
+    const val WEIGHER_ID = "kotlin.preferGetSetMethodsToProperty"
+    private var LookupElement.propertyName by UserDataProperty(Key<Name>("KOTLIN_PROPERTY_NAME"))
+
+    fun KtAnalysisSession.addWeight(lookupElement: LookupElement, symbol: KtSymbol) {
+        lookupElement.propertyName = (symbol as? KtPropertySymbol)?.name
+    }
+
+    object Weigher : LookupElementWeigher(WEIGHER_ID, false, true) {
+        override fun weigh(element: LookupElement, context: WeighingContext): Int {
```

Comment:
Maybe we can just use a `Boolean` instead of `Int` here?

(I see now that you just used the code from the old plugin, so we probably shouldn't - just to keep it similar)

Reply:
Yeah I was just trying to align to FE1.0. I actually like `Boolean` better, so let's do it!